test/python/result/test_mitigators.py

```diff
@@ -491,5 +427,109 @@ def test_error_handling(self):
         )
 
 
+class TestLocalReadoutMitigation(QiskitTestCase):
+    """Tests specific to the local readout mitigator"""
+
+    def test_assignment_matrix(self):
+        """Tests that the local mitigator generates the full assignment matrix correctly"""
+        qubits = [7, 2, 3]
+        assignment_matrices = [
+            np.array([[0.98828125, 0.04003906], [0.01171875, 0.95996094]]),
+            np.array([[0.99023438, 0.02929688], [0.00976562, 0.97070312]]),
+            np.array([[0.984375, 0.02441406], [0.015625, 0.97558594]]),
+        ]
```

Comment:
Again, it'd be good to pull these specific numbers directly from `FakeYorktown` in the code, so a code reader can more easily see what's going on, but if that's not possible, then a comment like on the previous one wouldn't hurt.

Reply:
Done, thanks.